pkg/arvo/sys/vane/clay.hoon

```diff
@@ -2765,12 +2883,19 @@
         $(aeon +(aeon))
       ::  produce updated state
       ::
+      =/  =rave  rave:(~(got by bom.u.ref) inx)
+      ?>  ?=(%many -.rave)
```

Comment:
Small point of style, but I think the diff would be clearer if these lines were added in the `lim` update expression below, since that's the only place they're used.